kcidb/db/__init__.py

```diff
@@ -8,32 +8,39 @@
 import kcidb.orm
 import kcidb.misc
 from kcidb.misc import LIGHT_ASSERTS
-from kcidb.db import abstract, schematic, mux, \
-    bigquery, postgresql, sqlite, json, null, misc  # noqa: F401
+from kcidb.db import (
+    abstract,
+    schematic,
+    mux,
+    bigquery,
+    postgresql,
+    sqlite,
+    json,
+    null,
+    misc,
+)  # noqa: F401
 
 # Module's logger
 LOGGER = logging.getLogger(__name__)
 
 
 class MuxDriver(mux.Driver):
-    """Kernel CI multiplexing database driver"""
+    """Kernel CI multiplexing database driver."""
 
     @classmethod
     def get_doc(cls):
-        """
-        Get driver documentation.
+        """Get driver documentation.
 
         Returns:
             The driver documentation string.
         """
-        return super().get_doc() + \
-            "\n            Example: postgresql bigquery:kcidb_01"
+        return super().get_doc() + "\n            Example: postgresql bigquery:kcidb_01"
 
     @classmethod
     def get_drivers(cls):
-        """
-        Retrieve a dictionary of driver names and types available for driver's
-        control.
+        """Retrieve a dictionary of driver names and types available for
+        
+        driver's control.
```

Comment:
Uh-oh, I don't think this would do. The first line is [supposed](https://peps.python.org/pep-0257/#multi-line-docstrings) to be a summary and the lines after it and the blank line - a more elaborate description. Just spilling the original summary over to the elaborate description misses the point. The right solution here should be at least shortening the original summary until it fits the new limit, and then posting the original summary as the elaborate description. Still nicer would be elaborating more there, but that's optional.

Same goes for similar solutions everywhere else.

Reply:
Actually here the pydodstyle was giving me error for adding a blank link after each description first line that's why I have to add it too and I also didn't liked it much.